.changeset/curly-buttons-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added a new `NotificationToast` component that provides quick and contextual feedback based on the outcome of an action.
```

Comment:
Not necessarily based on an action, see https://github.com/sumup-oss/circuit-ui/pull/1295#discussion_r766028569 🙂

Reply:
Maybe we can just align it with the docs

```suggestion
Added a new `NotificationToast` component that provides quick and contextual feedback to the user.
```